### PR TITLE
Exclude unit tests, examples and benchmarks from git archives

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/testunit export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
```
Comment (roxblnfk) [inline .gitattributes]: \r\n

Comment (simonschaufi) [reply .gitattributes]: you mean add new line?

Comment (roxblnfk) [reply .gitattributes]: yes

Comment (simonschaufi) [reply .gitattributes]: done

**Comment by roxblnfk** [comment]
what about `/benches`?

**Comment by simonschaufi** [comment]
I can add it as well. I think it makes sense. Just the examples coud also be added in the download, or what do you think? I woul exclude them as well as they are not productive code.

**Comment by roxblnfk** [comment]
I would also exclude them

**Comment by simonschaufi** [comment]
done

**Comment by simonschaufi** [comment]
are you able to merge this?

**Comment by roxblnfk** [comment]
> are you able to merge this?

No. I do not have [write access](https://help.github.com/articles/what-are-the-different-access-permissions)

**Pull request merged**